zigzag/module_hierarchy_facade.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+
+# ======================================================================================================================
+# Imports
+# ======================================================================================================================
+
+from __future__ import absolute_import
+import re
+
+
+class ModuleHierarchyFacade(object):
+
+    _TESTCASE_GROUP_RGX = re.compile(r'tests\.(test_[\w-]+)\.?(Test\w+)?$')
+
+    def __init__(self, testcase_xml, mediator):
+        """Create a ModuleHierarchyFacade
+
+        Args:
+            testcase_xml (ElementTree): A XML element representing a JUnit style testcase result.
+            mediator (ZigZag): the mediator that stores shared data
+        """
+        self._mediator = mediator
+        self._testcase_xml = testcase_xml
+
+    def get_module_hierarchy(self):
```

Comment:
Nit pick, this could be a property instead of a method. Not really that big of a deal.